concent_api/verifier/tasks.py

```diff
@@ -36,6 +35,12 @@
 from .utils import unpack_archive
 
 
+# OpenCV is imported conditionally because it requires system module to work,
+# which is not installed on Concent instances without verifier.
+if 'verifier' in settings.CONCENT_FEATURES:
```

Comment:
isn't local importing the better approach?

Reply:
Could be, the only advantage of current solution which I see is that import remains on top of the file. But generally removing `if` is also an advantage. So as you wish, I don't have any strong opinion on this one.

Reply:
I think it's good enough as is. I don't have a strong preference either way but having it on top of the file is indeed a bit more informative - you can see at a glance that the module is using OpenCV.